### DISCUSSION TITLE
Slow operation of services in the LAN

### DISCUSSION THREAD

**Discussion by enzzo-ux** [strongswan/strongswan#543] [General]
Hi
We changed the server from opensuse to xen opensuse tumbleweed.
We moved the configuration from the old server that was running and generated the certificates. The connection between the client (openuse strongswam) and the strongswam server works.
(STRONGSWAN) WAN IP - SERVER - LAN IP (HTTP, SSH)
There is one client that gets ip from LAN pool when connected to strongswan.
But there is one problem, a short one was not on the previous server:
ICMP: Works for all hosts on LAN with 1360 packet
WEB: does not work for hosts in LAN and server (telnet ip_host_lan 80 works),
SSH: Works, but running eg mc or iperf in shell crashes the console.
If I change leftsubnet to 10.0.30.10/32, HTTP on the server opens quickly and ssh is fully functional,
however, I cannot communicate with other hosts on the LAN.
The firewall is now a firewalld, there was iptables on the previous server

conn SERVER_VPN
    auto = add
    keyexchange = ikev2
    left =% defaultroute
    left =% any
    leftid = 200.x.x.x - server_ip_address
    leftcert = /etc/ipsec.d/certs/server.cert.pem
    leftauth = pubkey
    leftsubnet = 10.0.30.0/24
    right =% any
    rightcert = /etc/ipsec.d/certs/client.cert.pem
    rightsourceip = 10.0.30.10 - client_ip_address
    rightauth = pubkey
    fragmentation = yes
    dpdaction = clear
    dpdtimeout = 600
    dpddelay = 60
    forceencaps = yes

conn CLIENT_VPN
    auto = start
    keyexchange = ikev2
    left =% defaultroute
    leftsourceip =% config
    leftcert = /etc/ipsec.d/certs/client.cert.pem
    leftauth = pubkey
    right = 200.x.x.x - server_ip_address
    rightcert = /etc/ipsec.d/certs/server.cert.pem
    righsubnet = 10.0.30.0/24
    rightauth = pubkey
    fragmentation = yes
    dpdaction = restart
    dpdtimeout = 600
    dpddelay = 60
    closeaction = restart
    keyintegries =% forever 

the result of tcpdump when trying to connect to local http :

09:31:57.069330 IP 10.0.3.10.59284 > sun.http: Flags [S], seq 3483463449, win 64800, options [mss 1440,sackOK,TS val 470979916 ecr 0,nop,wscale 7], length 0 
09:31:57.071247 IP 10.0.3.10.59286 > sun.http: Flags [S], seq 415687483, win 64800, options [mss 1440,sackOK,TS val 470979918 ecr 0,nop,wscale 7], length 0 
09:31:57.072358 IP 10.0.3.10.59284 > sun.http: Flags [.], ack 1775836331, win 507, options [nop,nop,TS val 470979919 ecr 28021019], length 0 
09:31:57.073510 IP 10.0.3.10.59284 > sun.http: Flags [P.], seq 0:563, ack 1, win 507, options [nop,nop,TS val 470979920 ecr 28021019], length 563: HTTP: GET / HTTP/1.1 
09:31:57.148437 IP 10.0.3.10.59286 > sun.http: Flags [.], ack 2484005949, win 507, options [nop,nop,TS val 470979986 ecr 28021021], length 0 
09:31:57.529064 IP 10.0.3.10.59284 > sun.http: Flags [.], ack 1, win 507, options [nop,nop,TS val 470980376 ecr 28021023,nop,nop,sack 1 {2741:2865}], length 0 
09:31:57.529064 IP 10.0.3.10.59284 > sun.http: Flags [.], ack 1, win 507, options [nop,nop,TS val 470980376 ecr 28021023,nop,nop,sack 2 {2741:2865}{2741:2865}], length 0 
09:32:12.363538 IP 10.0.3.10.59284 > sun.http: Flags [.], ack 1, win 507, options [nop,nop,TS val 470995209 ecr 28021023,nop,nop,sack 1 {2741:2866}], length 0 
09:32:28.653980 IP 10.0.3.10.59286 > sun.http: Flags [.], ack 1, win 507, options [nop,nop,TS val 471011500 ecr 28021021], length 0 
09:32:38.674803 IP 10.0.3.10.59286 > sun.http: Flags [.], ack 2, win 507, options [nop,nop,TS val 471021521 ecr 28062614], length 0

**Comment by tobiasbrunner** [comment]
Maybe have a look at [this wiki page](https://wiki.strongswan.org/projects/strongswan/wiki/ForwardingAndSplitTunneling#MTUMSS-issues).

**Comment by enzzo-ux** [comment]
The firewalld rules did not solve the problem.I just don't understand 
the rules of this firewall and have no desire to learn again :)

Changing the MTU to 1360 in the kernel-netlink.conf file in charon fixed 
the problem.


W dniu 10.08.2021 o 18:26, Tobias Brunner pisze:
